olm/quickstarts/rhosak-openshift-quarkus-bind-quickstart.yaml

```diff
@@ -3,223 +3,230 @@ kind: ConsoleQuickStart
-        1. Execute the command `oc get deployment`. This should returen the **Deployment** of you Quarkus application, and the **Deployment** of your tools container (if you're using that container).
+           ```
+           $ oc project my-project
+           ```
 
-      review:
-        instructions: |-
-          #### Verify that you've successfully connected your `oc` client to your OpenShift instance:
-          Did you see the **Deployment** of your Quarkus application as output of the `oc get deployment` command?
-        failedTaskHelp: This task isn’t verified yet. Try the task again.
-      summary:
-        success: >-
-          Great work! You have access to the tooling required for the remainder of this quickstart.
-        failed: Try the steps again.
-    - title: Bind your Quarkus application to your Streams for Apache Kafka instance.
-      description: |-
-        With your Quarkus application running, and your Streams for Apache Kafka instance connected to your namespace, you can now bind your application to your Kafka instance.
-        This is done using the Service Binding Operator, which will inject the configuration values required to connect to your Kafka instance into your Quarkus application.
-        The Quarkus application has been configured to use the `quarkus-kubernetes-service-binding` [extension](https://github.com/DuncanDoyle/quarkus-kafka-quickstart-summit2021/blob/main/pom.xml#L82-L85)
-        enabling auto-discovery of the binding files injected into the Quarkus application pod.
+        1. Use the RHOAS CLI to bind your Kafka instance to your OpenShift project.
+
+           ```
+           $ rhoas cluster bind
+           ```
+
+           You're prompted to specify the Kafka instance in Streams for Apache Kafka that you want to connect to OpenShift.
 
-        We will create the binding using the `rhoas` command line tool. In future versions of OpenShift, you will also be able to do this directly from the OpenShift Developer Console.
+        1. Type the name of the Kafka instance that you want to connect to OpenShift. Press **Enter**.
 
-        To create the binding, we simply use the `rhoas cluster bind` command, and select the application deployment that we want to bind to our Streams for Apache Kafka instance that has already been connected to our OpenShift project.
+           You're prompted to specify the OpenShift application that you want to bind your Kafka instance to.
 
-        1. Go to the terminal of your tools container (or your own terminal if are using the tooling from your own machine).
+        1. To bind to the the example Quarkus application that you deployed, type `rhoas-quarkus-kafka-quickstart`. Press **Enter**.
```

Comment:
Same here, I AFAIK you can simply select the application you want to bind to from a list.

Reply:
See previous comment.